src/test/java/com/exasol/adapter/jdbc/JDBCAdapterIT.java

```diff
@@ -27,14 +26,14 @@ void testRegisteredAdapterFactories() throws AdapterException {
                 + "        \"name\" : \"foo\",\n" //
                 + "        \"properties\" :\n" //
                 + "        {\n" //
-                + "            \"" + SQL_DIALECT_PROPERTY + "\" : \"DERBY\"\n," //
                 + "            \"" + CONNECTION_NAME_PROPERTY + "\" : \"DERBY_CONNECTION\"\n" //
                 + "        }\n" //
                 + "    }\n" //
                 + "}";
         final ExaMetadata exaMetadata = Mockito.mock(ExaMetadata.class);
         RequestDispatcher.adapterCall(exaMetadata, rawRequest);
-        final List<AdapterFactory> registeredFactories = AdapterRegistry.getInstance().getRegisteredAdapterFactories();
-        assertThat(registeredFactories, hasItem(instanceOf(JDBCAdapterFactory.class)));
+        final ServiceLoader<AdapterFactory> adapterFactoryLoader = ServiceLoader.load(AdapterFactory.class);
```

Comment:
this is OK for now, eventually we have to implement a more thorough integration test. Please create a ticket for it if there.

Reply:
Done https://github.com/exasol/virtual-schema-common-jdbc/issues/96